packages/hooks/src/createCurrentHook.ts

```diff
@@ -1,14 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import type { Hooks, StoreKey } from '.';
```

Comment:
Let us import these from the `types.ts` file

```suggestion
import type { Hooks, StoreKey } from './types';
```